lld/test/ELF/aarch64-got-relocations-pauth.s

```diff
@@ -77,6 +77,60 @@ _start:
   adrp x1, :got_auth:zed
   add  x1, x1, :got_auth_lo12:zed
 
+#--- ok-tiny.s
+# RUN: llvm-mc -filetype=obj -triple=aarch64 ok-tiny.s -o ok-tiny.o
+
+# RUN: ld.lld ok-tiny.o a.so -pie -o external-tiny
+# RUN: llvm-readelf -r -S -x .got external-tiny | FileCheck %s --check-prefix=EXTERNAL-TINY
```

Comment:
I might just use tiny1 tiny2 instead of local-tiny external-tiny, since we may test both preemptible and non-preemptible symbols.    local/external isn't accurate linker term anyway (local may mean the local binding)

Reply:
Thanks for suggestion, fixed in 7d949458a8c0ce85e52f558159298090fa3bf529, and also applied the same to #113815